app/views/transportMeans/departure/RemoveDepartureMeansOfTransportYesNoView.scala.html

```diff
@@ -0,0 +1,55 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import controllers.transportMeans.departure.routes
+@import models.LocalReferenceNumber
+@import viewModels.components.InputYesNoViewModel._
+@import viewModels.components.InputYesNoViewModel
+@import views.html.components.{Button, ErrorSummary, InputYesNo, InsetText}
+
+@this(
+    mainTemplate: MainTemplate,
+    formHelper: FormWithCSRF,
+    inputYesNo: InputYesNo,
+    govukInsetText : GovukInsetText,
+    errorSummary: ErrorSummary,
+    button: Button
+)
+
+@(form: Form[Boolean], lrn: LocalReferenceNumber, mode: Mode, departureIndex: Index,  insetText: Option[String])(implicit request: Request[_], messages: Messages)
+
+@mainTemplate(
+    title = messages("transportMeans.departure.removeTransportMeansOfDepartureYesNo.title", departureIndex.display),
+    showBackLink = true,
+    lrn = lrn
+) {
+
+    @formHelper(action = routes.RemoveDepartureMeansOfTransportYesNoController.onSubmit(lrn, mode, departureIndex), Symbol("autoComplete") -> "off") {
+
+        @errorSummary(form.errors)
+
+        @inputYesNo(
+            field = form("value"),
+            yesNoType = InputYesNoViewModel(
+                heading = messages("transportMeans.departure.removeTransportMeansOfDepartureYesNo.heading", departureIndex.display),
+                caption = Some(messages("departureMeansOfTransport.caption")),
+                additionalHtml = insetText.map(x => govukInsetText(InsetText(content = Text(x))))
```

Comment:
If you rename insetText to something like `departureMeansOfTransport: String` and change `govukInsetText : GovukInsetText` back to `insetText: InsetText` then this line can just be `departureMeansOfTransport.map(insetText(_))`

Reply:
Done